src/iris/io/errors.py

```diff
@@ -45,6 +45,20 @@ class PupilIrisPropertyEstimationError(Exception):
 
     pass
 
+class Pupil2IrisValidatorError_Dilation(Exception):
+    """IsMaskGoodValidator error class."""
+
+    pass
+
+class Pupil2IrisValidatorError_Constriction(Exception):
+    """IsMaskGoodValidator error class."""
+
+    pass
+
+class Pupil2IrisValidatorError_Offcenter(Exception):
+    """IsMaskGoodValidator error class."""
+
+    pass
```

Comment:
Can we remove `_` from those errors' names to keep consistency in the code base? Also docstrings must be adjusted 😉.
```suggestion
class Pupil2IrisValidatorErrorDilation(Exception):
    """Pupil2IrisValidatorErrorDilation error class."""

    pass

class Pupil2IrisValidatorErrorConstriction(Exception):
    """Pupil2IrisValidatorErrorConstriction error class."""

    pass

class Pupil2IrisValidatorErrorOffcenter(Exception):
    """Pupil2IrisValidatorErrorOffcenter error class."""

    pass
```

Reply:
fixed, thanks!